src/components/Cards/CardsContainer.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import { Component } from "react";
+
+import ApiSubject from "../../../api/mockedApi";
+import { ICard } from "../../../api/mockedResponse";
+
+import Card from "./Card";
+
+export default class CardsContainer extends Component<any, { cards: ICard[] }> {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cards: [],
+    };
+  }
+
+  componentDidMount() {
+    ApiSubject.subscribe(this.updateCardsState);
+    ApiSubject.dispatch();
+  }
+
+  componentWillUnmount() {
+    ApiSubject.unsubscribe(this.updateCardsState);
+  }
+
+  updateCardsState = (cards: ICard[]) => this.setState({ cards });
+
+  renderCards = () => {
+    const { cards } = this.state;
+    if (!cards || !cards.length) {
+      return <div>No cards yet</div>;
+    }
+    return cards.map(({ gender, id, imageUrl, price, title }) => {
```

Comment:
What do you think about make it this way and then use card={card}?
```suggestion
    return cards.map((card: ICard) => {
```

Reply:
Fixed
https://github.com/BlackTruth/react/blob/5e20841788b32724609b7a987dbab75974a9c0d3/src/components/Cards/CardsContainer.tsx#L33-L37